### DISCUSSION TITLE
New Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by outdoorbits** [outdoorbits/little-backup-box#244] [General]
The Raspberry Pi 5 should be available from the end of October 2023. The specifications sound very promising:

- The two USB3 ports can now be used at the same time with full bandwidth.
- The power supply to the USB ports has been significantly improved!
- The computing power has been significantly improved.
- The read and write speeds of fast SD cards can be better utilized.
- NVMe SSDs can be connected. (You'll probably need an adapter for this, but I don't know yet whether it'll make sense to use it.)
- There is a real time clock. With an optional battery, the time is not lost every time you reboot. (I still lack precise information about the power supply.)

The only disadvantage I see at the moment is that the new Raspberry Pi also consumes more energy and requires a 5V/25W power supply. Furthermore, it is not yet clear whether the current fan solution will still work.

I'm already working on customizing the case: https://github.com/outdoorbits/case-for-little-backup-box/tree/main/Raspberry_Pi_5
On the software side, so far I only see the NVMe SSDs as a necessary adjustment.

Please share your insights here to speed up the adaptation!

**Comment by shreblov** [comment]
I got an Pi 5 just now and have issues with the installation. When I use the Raspberry Imager and select the Pi 5, I only get Bookworm OS's. When downloading the latest BullsEye manually, the Pi 5 does not boot.

**Comment by outdoorbits** [comment]
You can use bookworm, LBB should install and work fine but Comitup is not yet compatible. Up to now you have to select not to install Comitup. 
The maintainer of Comitup has to rewrite his code replacing central librarys. He says, this will take months.

**Reply by shreblov** [reply]
I installed Bookworm Lite 32bit and run the installation without Comitup. After reboot the led on the Pi 5 is flashing red/green. This is the content of the install-error.log:
http://raspbian.raspberrypi.com/raspbian/dists/bookworm/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
Repository 'http://archive.raspberrypi.com/debian bookworm InRelease' changed its 'Suite' value from 'testing' to 'stable'
Synchronizing state of openvpn.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable openvpn
Removed "/etc/systemd/system/multi-user.target.wants/openvpn.service".
rm: cannot remove '/home/reiner/little-backup-box': No such file or directory
Cloning into 'little-backup-box'...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4707  100  4707    0     0  19227      0 --:--:-- --:--:-- --:--:-- 19212
Updating index cache for path `/usr/local/man/man1'. Wait...
mandb: warning: /usr/local/man/man1/rclone.1: whatis parse for rclone(1) failed
done.
2023/11/19 11:06:30 ERROR : Error reading tag file at /root/.cache/rclone/webgui/tag 
2023/11/19 11:06:30 NOTICE: A new release for gui (v2.0.5) is present at https://github.com/rclone/rclone-webui-react/releases/download/v2.0.5/currentbuild.zip
2023/11/19 11:06:30 NOTICE: Downloading webgui binary. Please wait. [Size: 4763452, Path :  /root/.cache/rclone/webgui/v2.0.5.zip]
2023/11/19 11:06:31 NOTICE: Unzipping webgui binary
2023/11/19 11:06:31 NOTICE: Serving Web GUI
2023/11/19 11:06:31 NOTICE: Serving remote control on http://[::]:5572/
2023/11/19 11:06:31 NOTICE: Web GUI is not automatically opening browser. Navigate to http://lbb:14bd3e8e23e8db4ca0a4@[::]:5572/?login_token=bGJiOjE0YmQzZThlMjNlOGRiNGNhMGE0 to use.
no crontab for reiner
no crontab for reiner
http://raspbian.raspberrypi.com/raspbian/dists/bookworm/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
rm: cannot remove './tinyfilemanager': No such file or directory
Cloning into 'tinyfilemanager'...
.....+.+......+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*......+......+.......+.....+...+...+.........+..........+...+..+.+.....+.........+.............+..............+....+.........+.....+.+...........+......+.......+.....+.+...........+...............+....+......+...........+...+..........+...........+...............+...+.............+..+...+...+..........+.............. (and a few more lines with this +...+)

**Reply by shreblov** [reply]
Strange, I tried again with the 64bit Bookworm Lite. Got the same error-log as above, but if I enter https://ip-of-the-pi I get the LLB up and running. LED is still flashing red/green.

**Comment by outdoorbits** [comment]
The error log looks ok. Some installation scripts write information to error, I didn't see any installation problem. The behaviour of the leds is not completely surprising. LEDs changed not the first time in RPi's history. I'm still hoping to get a Pi 5 but actually it seems not to be anywhere in stock. As soon I got mine, I'll adapt the LEDs. Do the backup functions work? Did you try View? I'm very excited about the speed of image converting.

**Reply by shreblov** [reply]
I want to backup videos taken with an action camera during scuba diving abroad. And I think without Comitup it won’t work for me. I did a backup and it worked but not as intended. It copied from the SSD to the SD card instead of otherway round. Inam missing a functionality to select what the source is and what the destination is.

**Comment by outdoorbits** [comment]
It's easy, just connect the target first.

**Comment by outdoorbits** [comment]
There is another idea: If you want to access the web UI without unsing 
comitup, you could setup a hotspot from your mobile phone. In 
installation process, you can configure this wifi in Raspberry Pi 
Imager. The RPi will connect to your hotspot whenever it is available. 
You just need to find the IP of your Pi then.

**Reply by shreblov** [reply]
Did some more tries. First I installed Bookworm Lite 32 and set via sudo raspi-config>Advanced Wayland back to X11 and installed LLB with Comitup. Installation was completed but Comitup did not start. Same with Lite 64. Seems to be more issues with Comitup than just Wayland.
Than I installed a fresh Bookworm Lite 64 and installed LLB without Comitup. Than I followed exactly [this description](https://gist.github.com/narate/d3f001c97e1c981a59f94cd76f041140) to configure a Hotspot on the Pi 5. Now I can connect my phone to the Hotspot and then to 10.42.0.1 and LBB is working.
I am still struggling with the process to get target and source correctly mounted. And I need to do some testing.

**Comment by outdoorbits** [comment]
If the are problems about your backup direction, just disconnect source before starting backup and wait until you get the introduction to connect source by display or log monitor.

**Comment by outdoorbits** [comment]
Are you using lite or a graphical surface? Please don't use LBB in a graphical environment. This might manage devices itself and block the mount functions of LBB.

**Comment by shreblov** [comment]
The first SSD I used as target was faulty, also a memory stick which I used as second target. With a new SSD and understanding how the plugin of target and source is working, all good. Transfer speed is around 80 MB per second, which is what the MicroSD as source can deliver. Generation of thumbnails from mp4 takes roundabout 10 seconds per file.
